pkg/compilers/interface.go

```diff
@@ -16,6 +16,10 @@ type Compiler interface {
 }
 
 type CompilerUsage struct {
+	// GeneralDescription section briefly describes what user has
+	// to prepare.
+	GeneralDescription string
```

Comment:
Temptation to implement this tiny upgrade here inline was simply to strong😄
Thanks for pointing out, let me do this properly this time.